CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - build(ios): Bump sentry-cocoa to 6.1.4 #1308
 - fix: Handle auto session tracking start on iOS #1308
+- ref: Stop tracking sample Podfile.lock #1310
```

Comment:
```suggestion
```

changelog is for the users, let's do it only when it's important.

Reply:
Ah I see, thanks!